.travis.yml

```diff
@@ -1,18 +1,27 @@
 language: python
 
-python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-  - "3.6"
-
 # command to install dependencies
-install: "python setup.py install"
+install:
+    - "python setup.py install"
 
 # command to run tests
-script: 'python setup.py test'
+script:
+    - python -m pytest -v
```

Comment:
running this required me to `pip install mock` before it would work. which is fine, assuming that’s available inside travis; if not, maybe we could include it in that setup script.

also worth mentioning that `python setup.py test` worked for me out-of-the-box on this branch

Reply:
Hmm, `python setup.py install` should get everything necessary, I'll look into this later.